.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+
+updates:
+
+  # Keep dependencies for GitHub Actions up-to-date
+  - package-ecosystem: 'github-actions'
```

Comment:
In the [nim-test-runner](https://github.com/exercism/nim-test-runner/blob/5c1a070ca424a77fd8923aba667428438e30c0f0/.github/dependabot.yml#L11) there is also a `docker` ecosystem for dependabot PRs. Can you open an issue to look in to this or add it in here if you think it's useful?

Reply:
Docker won't be useful here as there is no dockerfile in this repo. You can always update this file when a Dockerfile is added . For now this is sufficient.